update.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
I'd recommend ([via](http://redsymbol.net/articles/unofficial-bash-strict-mode/)):

```suggestion
set -eou pipefail
```